### ISSUE TITLE
Crash at id screen

### ISSUE THREAD

**Issue by mgood7123** [raing3/tempar#5]
I have compiled with these changes https://github.com/mgood7123/tempar-pspdev-latest/commit/7e0d730e131d223a7cf468787c8223503ac07723 for latest psp sdk

If we go to the id screen

![20220927_160117](https://user-images.githubusercontent.com/26642847/192445327-b3ced911-87b7-42b6-b241-9a1c9b7fe013.jpg)

Then tempar crashes and we cannot quit nor change screens

We need to power off to exit

Compared to the original tempar

![20220927_155914](https://user-images.githubusercontent.com/26642847/192445003-b67193c0-0e29-4d74-a5de-101cf688575c.jpg)

**Comment by mgood7123** [comment]
Compared to the original tempar

![20220927_155914](https://user-images.githubusercontent.com/26642847/192445003-b67193c0-0e29-4d74-a5de-101cf688575c.jpg)

**Comment by mgood7123** [comment]
Fixed in https://github.com/mgood7123/tempar-pspdev-latest/commit/3702b5abab70e5b163b65c21775649ff069b7a00

**Comment by raing3** [comment]
Thanks very much, I've added that change to the repo + fixed some build issues I encountered (you also look to have done similar fixes on your fork). Appreciate the effort in identifying/resolving the issue.

Prior to making the source code public way back I removed some xor string transformations done on the credits and think I must have removed the return in the process.

Hopefully I can work out why the special character font patching isn't working properly when compiling. :D

If you end up making any other changes you think would be beneficial feel free to submit a PR as well if you would like. I'd love to do more on the repo myself but among everything else I want to do I struggle to find the time. :)
